Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCheckPointFirewall.yaml

```diff
@@ -0,0 +1,260 @@
+let NWParser=(disabled:bool=false){
+    CommonSecurityLog
+    | where not(disabled)
+    | where DeviceVendor=="Check Point" and DeviceProduct=="VPN-1 & FireWall-1"
+    | extend 
+            EventProduct = "Firewall",
+            EventCount = toint(1),
+            EventType = "NetworkSession",
+            EventSchema = "NetworkSession",
+            EventSchemaVersion = "0.2.4"
+    | parse AdditionalExtensions with * "rule_action=" DeviceAction ";" * "rule_uid=" NetworkRuleName ";" *
+    | parse AdditionalExtensions with * "loguid=" EventOriginalUid ";" * "origin=" Dvc ";" * "originsicname=CN\\=" DvcHostname "," *
+    | parse AdditionalExtensions with * "inzone=" SrcZone ";" * "outzone=" DstZone ";" * 
+    | parse AdditionalExtensions with * "conn_direction=" CommunicationDirection ";" 
+    | parse AdditionalExtensions with * "alert=" AlertName ";" *
+    | parse AdditionalExtensions with * "inspection_category=" InspectionCategory ";" * "inspection_item=" ThreatName ";"                   
+    | extend
+            ThreatCategory = coalesce(AlertName, InspectionCategory),
+            EventResult = case(
+                                isempty(DeviceAction), "", 
+                                DeviceAction == "Accept", "Success",
+                                "Failure"
+                            ), 
+            NetworkRuleName = coalesce(DeviceCustomString2, NetworkRuleName),
+            EventStartTime = unixtime_milliseconds_todatetime(tolong(ReceiptTime))
+    | project-rename  
+            EventVendor = DeviceVendor,
+            DstPortNumber = DestinationPort,
+            DstIpAddr = DestinationIP,
+            SrcPortNumber = SourcePort,
+            SrcIpAddr = SourceIP,
+            DstNatIpAddr = DestinationTranslatedAddress,
+            DstNatPortNumber = DestinationTranslatedPort,
+            SrcNatIpAddr = SourceTranslatedAddress,
+            SrcNatPortNumber = SourceTranslatedPort,
+            EventProductVersion = DeviceVersion,
+            EventOriginalSeverity = LogSeverity,
+            Rule = NetworkRuleName,
+            DstAppName = Activity,
+            DvcOriginalAction = DeviceAction
+    | lookup ProtocolLookup on Protocol
+    | lookup DirectionLookup on CommunicationDirection
+    | lookup ActionLookup on DvcOriginalAction
+    | extend 
+            EventEndTime = EventStartTime,
+            IpAddr = SrcIpAddr,
+            Dst = DstIpAddr,
+            Src = SrcIpAddr,
+            EventMessage = Message,
+            EventSeverity = case(
+                                DvcAction == "Drop", "Low",
+                                DvcAction == "Allow", "Informational",
+                                ""
+                            ),
+            NetworkDirection = case(
+                                isnotempty(NetworkDirection), NetworkDirection,
+                                SrcZone == "Internal" and (DstZone == "Internal" or DstZone == "Local"), "Local",
```

Comment:
I think you skipped my comment on DstZone and SrcZone. You should not map them but only use the parsed values ot determine NetworkDirection.

Reply:
I commented a question on the previous PR. Sorry for multiple locations, posting it here - 

**Sample values are Internal, External, Local. Should we consider them as Interfaces?** 

If yes, should we include watchlist here as well?

Reply:
As documented, the values (inzone, outzone) are not interfaces or zones, but indications whether the source (inzone) or destinaion (outzone) networks are internal or external. As such it should be used only to determine NetworkDirection and not set either zones or interfaces.

Reply:
Understood. I found one more column "CommunicationDirection" which can be used to enrich Direction details. Replacing the logic to reduce complexity.

Reply:
Fine as is.